database/__main__.py

```diff
@@ -0,0 +1,31 @@
+from database.connector import Connector
+
+
+if __name__ == '__main__':
+    connector = Connector()
+    #connector.drop_all_tables()
+    connector.clean_all_tables()
+    event_id1 = connector.save_event_info("sochi")
+    event_id2 = connector.save_event_info("ochi")
+    event_id3 = connector.save_event_info("sochi")
+    user_id1 = connector.save_user_info("boris", event_id1)
```

Comment:
We need to discuss what `user_id` is
Telegram provides `user.id` we may want to use, so that we might use Telegram's data as primary key instead of autoincrement field

Reply:
I think we also should have our custom user_id so same user can have different names in differen events